website/repairnator-mongo-rest-api/server/controllers/pipeline-error.controller.js

```diff
@@ -17,10 +17,10 @@ function get(req, res) {
 }
 
 /**
- * Get user list.
+ * Get pipeline-error list.
  * @property {number} req.query.skip - Number of users to be skipped.
  * @property {number} req.query.limit - Limit number of users to be returned.
```

Comment:
Might be a bit too picky, but I assume the comments are no longer relevant?

Reply:
It is the documentation.
I guess it is here because Simon copied most of the code from another project (APIs in JS is a lot of boiler plates).
But I guess it is relevant to the doc in the sourcecode.

Reply:
Okay, should probably be changed, but it's fine.